### PR TITLE
Update Section Label to use correct colour in secondary column

### PR DESCRIPTION
**Pull request by FK78**
Part of https://github.com/bbc/simorgh/issues/6587

**Overall change:** _Update Section Label to use correct colour in secondary column._

**Code changes:**

- _Remove `color` from `Title` so that it inherits colour from `SectionLabelLink` or `SectionLabelWrapper`._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by andrewscfc — APPROVED** [review]
Looking good on chronmatic:
image

Very neat change 🔍 

You may want to alter the PR so it doesn't close bbc/simorgh#6587 when you merge this, we still need to integrate this with a simorgh PR.

**Review by HarveyPeachey — APPROVED** [review]
LGTM

File: packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -53,6 +52,7 @@ exports[`SectionLabel When hideSectionHeader is true should add styling to hide
 .c0 {
   position: relative;
   z-index: 0;
+  color: #3F3F42;
```
Comment (simonsinclair) [inline packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap:55]: I can't seem to see the other colour listed in this snapshots. Do we have a test case for it?

**Review by simonsinclair — CHANGES_REQUESTED** [review]

File: packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (FK78) [reply packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap:55]: I can see `C_EBON` being used [here](https://github.com/bbc/psammead/blob/b66faf72639cc1b5fe11d46fe48e16589e91936c/packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap#L155).

**Comment by FK78** [comment]
Approved in UX Review ✅

File: packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (simonsinclair) [reply packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap:55]: Thanks, Fahad!

**Review by simonsinclair — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
LGTM...

**Pull request merged**